api/src/chat/dialogs/myanmar/disaster-assessment/questions/askUserInfo.ts

```diff
@@ -24,4 +24,209 @@ export const askUserInfoDialog = rcdaChatDialog(
             session.conversationData.mm.geographicalSettingId = getKeys(localizer.mm.geographicalSettings)[response.index] as MyanmarGeographicalSettings;
             session.endDialog();
         }
-    ]);
\ No newline at end of file
+    ],
+    {
+        references: () => [
+            confirmUserAdminStack
+        ]
+    });
+
+export const confirmUserAdminStack = rcdaChatDialog(
+    "/confirmUserAdminStack",
+    null,
+    [
+        ({ session, localizer, next }) => {
+            const currentAdminStack: string = session.userData.adminStack;
+            if (currentAdminStack !== undefined) {
```

Comment:
Let's keep the condition checks for this value consistent. On line 54 you do a general truthiness evaluation, instead of a direct value comparison against undefined. I generally prefer the below approach.